src/bloodhound.yml

```diff
@@ -1,17 +1,34 @@
 ---
 - name: Install and configure Neo4j
   block:
+    # By default OpenJDK 17 is being used on Kali Linux.  Neo4j <5
+    # requires OpenJDK 11.
+    #
+    # TODO: This block can be removed once we upgrade to Neo4j version
+    # 5, as discussed in #136.
+    - name: Install and configure Debian to use OpenJDK 11
+      block:
+        - name: Install OpenJDK 11
+          ansible.builtin.import_role:
+            name: openjdk
+        - name: Configure Debian to use OpenJDK 11
+          community.general.alternatives:
+            name: java
+            path: /usr/lib/jvm/java-11-openjdk-amd64/bin/java
```

Comment:
Do we need to worry about any of the other common Java commands (`javac` and `javaw` off the top of my head)?

Reply:
I don't think so, but here is the complete list:
```
jar                            auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jar
jarsigner                      auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jarsigner
java                           auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/java
javac                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/javac
javadoc                        auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/javadoc
javap                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/javap
jcmd                           auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jcmd
jconsole                       auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jconsole
jdb                            auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jdb
jdeprscan                      auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jdeprscan
jdeps                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jdeps
jexec                          auto     /usr/lib/jvm/java-17-openjdk-amd64/lib/jexec
jfr                            auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jfr
jhsdb                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jhsdb
jimage                         auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jimage
jinfo                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jinfo
jlink                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jlink
jmap                           auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jmap
jmod                           auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jmod
jpackage                       auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jpackage
jps                            auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jps
jrunscript                     auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jrunscript
jshell                         auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jshell
jstack                         auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jstack
jstat                          auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jstat
jstatd                         auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/jstatd
keytool                        auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/keytool
rmiregistry                    auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/rmiregistry
serialver                      auto     /usr/lib/jvm/java-17-openjdk-amd64/bin/serialver
```

I'll think about this more tomorrow.

Reply:
In any event, whatever `community.general.alternatives` jazz we decide is required should probably be moved to [cisagov/ansible-role-openjdk](https://github.com/cisagov/ansible-role-openjdk).

Reply:
Please see cisagov/ansible-role-openjdk#41.